### PR TITLE
grid.get_lgr: Can do numbered lookup.

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by kudhru** [comment]
Thanks!

File: python/python/ecl/ecl/ecl_grid.py [hunk 1]
```diff
@@ -78,7 +79,8 @@ class EclGrid(BaseCClass):
     _cell_contains                = EclPrototype("bool ecl_grid_cell_contains_xyz1( ecl_grid , int , double , double , double )")
     _cell_regular                 = EclPrototype("bool ecl_grid_cell_regular1( ecl_grid , int)")
     _num_lgr                      = EclPrototype("int  ecl_grid_get_num_lgr( ecl_grid )")
-    _has_lgr                      = EclPrototype("bool ecl_grid_has_lgr( ecl_grid , char* )")
+    _has_numered_lgr_             = EclPrototype("bool ecl_grid_has_lgr_nr( ecl_grid , int)")
```
Comment (markusdregi) [inline python/python/ecl/ecl/ecl_grid.py]: Should be _has_numbered_lgr?

Comment (joakim-hove) [inline python/python/ecl/ecl/ecl_grid.py]: Thank you - yes.

**Comment by joakim-hove** [comment]
> it seems like the numbered version of the get_lgr-function is never tested. 

Yes - that is unfortunately correct - quite large job to create valid testdata. It is tested in the Statoil data (and that test had not been run .... ). Will look into it.

**Pull request merged**